shared/haml/onetrust_cookie_scripts.haml

```diff
@@ -19,8 +21,11 @@
 -# domain was given.
 - one_trust_domain = one_trust_domains.dig(domain, cookie_script_env)
 - if cookie_script_env != 'off' && one_trust_domain.present?
-  - auto_block_src = "https://cdn.cookielaw.org/consent/#{one_trust_domain}/OtAutoBlock.js"
-  %script{src: auto_block_src, type: 'text/javascript'}
-  %script{src: 'https://cdn.cookielaw.org/scripttemplates/otSDKStub.js', type: 'text/javascript', charset: 'UTF-8', 'data-domain-script' => one_trust_domain}
+  - base_url = onetrust_version == 'self_hosted' ? "/onetrust" : "https://cdn.cookielaw.org"
```

Comment:
Does this work on hourofcode.com or even code.org? During our testing, I thought the url had to be studio.code.org/onetrust/

Reply:
This works for hourofcode.com
DCDO flag set to 'self-hosted'. You can see one trust in the sources
<img width="1792" alt="Screenshot 2024-08-26 at 3 09 47 PM" src="https://github.com/user-attachments/assets/d9d79afa-a9c3-4bf4-8e6d-c142b638f380">

Will check for studio.code.org

Reply:
<img width="1792" alt="Screenshot 2024-08-26 at 4 13 10 PM" src="https://github.com/user-attachments/assets/d64dee26-b4bc-4634-933d-71658f781f96">
Heres studio.code.org